### DISCUSSION TITLE
GDPR regulations - Windows server receives identifiable information

### DISCUSSION THREAD

**Discussion by Mahdi89** [AxisCommunications/discussions#83] [General]
Hi there,
We are planning to deploy AXIS cameras in open areas in our city. We noticed that the Windows server which has device manager installed on it we can view raw identifiable data of citizens. Do you have any suggestions to better secure the server or eleminate the server after application deployment? 
Our concern is to understand how to fully secure our AI analytics platform so that our client (city council) is assured about GDPR compliance. 
Another option is to blurr faces in every image that is received by the camera. 
I'd greatly appreciate your help.
Best,
Mahdi

**Comment by mattias-kindborg-at-work** [comment]
Hi @Mahdi89!

Which device manager software are you referring to? Please provide the product name and possibly a link to the product.

**Reply by Mahdi89** [reply]
Hi @mattias-kindborg-at-work
We have downloaded the device manager from Axis portal directly https://www.axis.com/support/tools/axis-device-manager#download-block
Thanks

**Reply by mattias-kindborg-at-work** [reply]
Ok, got it. Now on to the next question. Can you elaborate what you mean with *it we can view raw identifiable data of citizens*? The server obviously has network access to the cameras, that cannot be the problem right?

**Reply by Mahdi89** [reply]
Thanks @mattias-kindborg-at-work  sorry for my delayed reply. This is still pneding a reply from our client.
